http/accesslog_handler.go

```diff
@@ -38,7 +38,7 @@ func (h accessLogHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 
 	defer func(begin time.Time) {
-		_ = h.logger.Log("method", req.Method, "uri", uri, "status_code", writer.Status(), "size", writer.Size(), "time", time.Since(begin).Milliseconds())
+		_ = h.logger.Log("method", req.Method, "uri", uri, "status_code", writer.Status(), "size", writer.Size(), "time_spent", time.Since(begin).Milliseconds())
```

Comment:
I am not a big fan of time_spent. it is kind of vague. Some suggestions : 
- request_time
- request_duration
- total_request_time
- processing_time
- execution_time
- elapsed_time